510_Deployment/40_config.asciidoc

```diff
@@ -1,53 +1,44 @@
 [[important-configuration-changes]]
-=== Important Configuration Changes
-Elasticsearch ships with _very good_ defaults,((("deployment", "configuration changes, important")))((("configuration changes, important"))) especially when it comes to performance-
-related settings and options.  When in doubt, just leave
-the settings alone.  We have witnessed countless dozens of clusters ruined
-by errant settings because the administrator thought he could turn a knob
-and gain 100-fold improvement.
+
+=== 重要配置的修改
+Elasticsearch 已经有了 _很好_ 的默认值，((("deployment", "configuration changes, important")))((("configuration changes, important")))特别是涉及到性能相关的配置或者选项。
+如果你有疑问，最好就不要动它。我们已经目睹了数十个因为错误的设置而导致毁灭的集群，
+因为它的管理者总认为改动一个配置或者选项就可以带来 100 倍的提升。
 
 [NOTE]
 ====
-Please read this entire section!  All configurations presented are equally
-important, and are not listed in any particular order.  Please read
-through all configuration options and apply them to your cluster.
+请阅读整节文章，所有的配置项都同等重要，和描述顺序无关，请阅读所有的配置选项，并应用到你的集群中。
 ====
 
-Other databases may require tuning, but by and large, Elasticsearch does not.
-If you are hitting performance problems, the solution is usually better data
-layout or more nodes.  There are very few "magic knobs" in Elasticsearch.
-If there were, we'd have turned them already!
+其它数据库可能需要调优，但总得来说，Elasticsearch 不需要。
+如果你遇到了性能问题，最好的解决方法通常是更好的数据布局或者更多的节点。
+在 Elasticsearch 中很少有“神奇的配置项”，
+如果存在，我们也已经帮你优化了。
 
-With that said, there are some _logistical_ configurations that should be changed
-for production.  These changes are necessary either to make your life easier, or because
-there is no way to set a good default (because it depends on your cluster layout).
+说到这里，有一些保障性的配置需要在生产环境中做修改。
```

Comment:
说到这里，有一些保障性的配置需要在生产环境中做修改。
这些变化是必要的，以便让你的生活更轻松，因为没有办法设定好的默认值（它取决于你的集群布局）。

或者

也就是说，有些配置在生成环境中是应该调整的。
这些变化是必要的，以便让你的生活更轻松，因为没有办法设定好的默认值（它取决于你的集群布局）。

怎么样。

2016-03-11 10:52 GMT+08:00 Medcl <notifications@github.com>:

> In 510_Deployment/40_config.asciidoc
> <https://github.com/elasticsearch-cn/elasticsearch-definitive-guide/pull/26#discussion_r55783942>
> :
>
> >
> > -With that said, there are some _logistical_ configurations that should be changed
> > -for production.  These changes are necessary either to make your life easier, or because
> > -there is no way to set a good default (because it depends on your cluster layout).
> > +说到这里，有一些保障性的配置需要在生产环境中做修改。
>
> 这样看行不行：也就是说，有些配置在生成环境中是应该调整的
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elasticsearch-cn/elasticsearch-definitive-guide/pull/26/files#r55783942>
> .
>